etl/steps/data/garden/papers/2023-12-12/nemet_2009.meta.yml

```diff
@@ -18,7 +18,9 @@ tables:
         unit: 2004 USD/Watt
       yearly_capacity:
         title: Capacity of photovoltaic technology
-        unit: MW
+        unit: megawatts
+        short_unit: MW
       previous_capacity:
         title: Previous capacity of photovoltaic technology
```

Comment:
again as you don't provide any detail in the metadata it's not clear what these indicators actually show - e.g., what's previous?